### PR TITLE
Improve the way email message is displayed 

### PR DESCRIPTION
**Pull request by xypnox**
I have implemented the following changes:

- Removed unnecessary icons. We do not need the icons everywhere and only content of email matters.
- Sent to is now in the footer in a less obtrusive position
- The color green sends a signal of being complete. This has been changed to blue to represent a notification rather than a success
- All important information is preserved: sender's email, send to email, timestamp, content etc.

Here is a screenshot for the new format of displaying emails:
screenshot_2019-01-26 random common ideas slack 2

### CODE REVIEW

**Comment by Ayushk4** [comment]
By mistake ^.

**Comment by xypnox** [comment]
Genius 101

**Comment by Ayushk4** [comment]
I wish I could react with :xypnox: here ^. :stuck_out_tongue:

**Comment by OrkoHunter** [comment]
Love this!

**Comment by OrkoHunter** [comment]
Enabled automatic deploys of the master branch on the Heroku app.

Note: The heroku app is hosted on KOSS account (So that it does not consume our free hours given by Heroku). However, I have added @xypnox and me as collaborators.

To enable GitHub-Heroku sync for deploys, it requires the connected GitHub account to have Admin access. We can not grant `koss-service` admin access to the org. Hence, the current Heroku uses my OAuth to do automatic deploys.

TLDR:
1. master branch is automatically deployed.
2. @koss-service GitHub account is supposed to have read only access to @kossiitkgp org.
